src/main/java/uk/gov/ons/census/casesvc/service/NewAddressReportedService.java

```diff
@@ -55,38 +54,28 @@ private Case createSkeletonCase(CollectionCase collectionCase) {
     skeletonCase.setLatitude(collectionCase.getAddress().getLatitude());
     skeletonCase.setLongitude(collectionCase.getAddress().getLongitude());
     skeletonCase.setUprn(collectionCase.getAddress().getUprn());
-    skeletonCase.setEstabUprn(collectionCase.getAddress().getEstabUprn());
     skeletonCase.setEstabType(collectionCase.getAddress().getEstabType());
     skeletonCase.setRegion(collectionCase.getAddress().getRegion());
-    skeletonCase.setActionPlanId(collectionCase.getActionPlanId()); // This is essential
```

Comment:
without an action plan ID, this case is effectively 'invisible' to Action Sched/Worker

Reply:
They appear happy with that, action_plan_id isn't copied from a source_case either.

Reply:
I'd want to confirm this, I hear 'action plan' being colloquially used a lot to refer to action rules and I'd want to be sure there was no confusion about that here. We're currently expecting to have all census cases under one action plan ID.

Reply:
action plan now set in yml